www/src/fonts/Webfonts/futurapt_book_macroman/stylesheet.css

```diff
@@ -24,6 +24,7 @@
 
 @font-face {
   font-family: "Futura PT";
+  font-display: swap;
```

Comment:
ah nice! This dropped the speed index a bit https://www.webpagetest.org/result/180914_AH_cff11777affacee5530bf49bb19ae111/

Reply:
Maybe ~200ms